src/components/Menu/menuRules.ts

```diff
@@ -6,12 +6,12 @@ const solidBorder = (color: string) => ({
 
 export default {
   root: ({ props, variables }) => {
-    const { type, shape, vertical } = props
+    const { fluid, type, shape, vertical } = props
     return {
       display: 'flex',
       ...(vertical && {
         flexDirection: 'column',
-        width: pxToRem(200),
+        ...(!fluid && { width: pxToRem(200) }),
```

Comment:
I think that a variable with this value should be good here.

Reply:
agree, let's address this problem by means of separate PR, here is an issue filed for that: https://github.com/stardust-ui/react/issues/62